docs/usage/nuget.md

```diff
@@ -49,6 +49,8 @@ In this example we defined 3 NuGet feeds.
 The package resolving process uses the `merge` strategy to handle the 3 feeds.
 All feeds are checked for dependency updates, and duplicate updates are merged/joined together into a single dependency update.
 
+If your project uses lockfiles (a `package.lock.json` exists), alternate feed settings are ignored and _only_ `NuGet.config` and `https://api.nuget.org/v3/index.json` are considered for lookups.
```

Comment:
```suggestion
If your project uses lockfiles (a `package.lock.json` exists), alternate feed settings must be defined in a `NuGet.config` as `registryUrls` are not passed through to the NuGet commands used.
```

Is this accurate?

Reply:
It is.

i'd want to add an "only" to make clear that central configuration will not work with lockfiles.